### PR TITLE
added README update workflow

### PR DESCRIPTION
**Pull request by 0b51d14n217**
added workflow to update README with repository contributors at each push.

### CODE REVIEW

**Comment by AbhijithGanesh** [comment]
@0b51d14n217 Please add a description to the Pull Request, Empty pull request is confusing often😅!

**Comment by 0b51d14n217** [comment]
> @0b51d14n217 Please add a description to the Pull Request, Empty pull request is confusing often😅!

Sure, updated Pull Request.

File: .github/workflows/readmeupdate.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  push:
+    branches:
+      - master
+
+jobs:
+  add-contributors:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: 0b51d14n217/Annual-Yearbook@readme-workflow
```
Comment (AbhijithGanesh) [inline .github/workflows/readmeupdate.yaml:14]:
```suggestion
 - run-name: Deploy to ${{ inputs.deploy_target }} by @${{ github.actor }})
```

Comment (AbhijithGanesh) [reply .github/workflows/readmeupdate.yaml:14]: This basically tells the action to run it in a particular manner!

File: .github/workflows/readmeupdate.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      with:
+        CONTRIBUTOR: 'https://github.com/ComputerSocietyVITC/Annual-Yearbook/graphs/contributors'
+        COLUMN_PER_ROW: '6'
+        ACCESS_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        IMG_WIDTH: '100'
+        FONT_SIZE: '14'
+        PATH: '/README.md'
+        COMMIT_MESSAGE: 'docs(README): update contributors'
+        AVATAR_SHAPE: 'round'
```
Comment (AbhijithGanesh) [inline .github/workflows/readmeupdate.yaml:23]: These are data points, we should rather focus on automating it! The COMMIT_MESSAGE is good though

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]

File: .github/workflows/readmeupdate.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: 0b51d14n217/Annual-Yearbook@readme-workflow
+ - run-name: Deploy to ${{ inputs.deploy_target }} by @${{ github.actor }})
```
Comment (AbhijithGanesh) [inline .github/workflows/readmeupdate.yaml:14]: You need to call it Change Readme by @$...
```suggestion
 - run-name: Changes to Readme - ${{ inputs.deploy_target }} by @${{ github.actor }})
```

**Comment by cool-pants** [comment]
@0b51d14n217 could you squash the last two commits to have a cleaner commit history?
Otherwise the content seems fine 👍.

File: .github/workflows/readmeupdate.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  push:
+    branches:
+      - master
+
+jobs:
+  add-contributors:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+ - run-name: Changes to Readme - ${{ inputs.deploy_target }} by @${{ github.actor }})
```
Comment (AbhijithGanesh) [inline .github/workflows/readmeupdate.yaml:14]: The indentation seems to be incorrect here😀 ! YAML is an indented file system!

Comment (AbhijithGanesh) [inline .github/workflows/readmeupdate.yaml:14]:
```suggestion
    - uses: actions/checkout@v2
    - run-name: Changes to Readme - ${{ inputs.deploy_target }} by @${{ github.actor }})
```

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]

**Comment by AbhijithGanesh** [comment]
Closed on request by @0b51d14n217

**Pull request closed without merging**